data-canary/scripts/globalevents/save_interval.lua

```diff
@@ -1,3 +1,14 @@
+local SAVE_INTERVAL_TYPE = configManager.getString(configKeys.SAVE_INTERVAL_TYPE)
```

Comment:
I don't see why moving this to the scripts if it's something shared... And you broke the naming convention.